example/provider-local/shoot-ipv6.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: local
+  namespace: garden-local
+  annotations:
+    shoot.gardener.cloud/infrastructure-cleanup-wait-period-seconds: "0"
+    shoot.gardener.cloud/cloud-config-execution-max-delay-seconds: "0"
+spec:
+  seedName: local
+  cloudProfileName: local
+  secretBindingName: local # dummy, doesn't contain any credentials
+  region: local
+  networking:
+    type: calico
+    pods: 2002:db8:1::/48
+#    nodes: 2002:db8:2::/64 -> it doesn't make sense to specify this, node IPs are given by the seed cluster's pod CIDR
+    services: 2002:db8:3::/108
+    ipFamilies:
+    - IPv6
+    # TODO(scheererj): Drop this once v1.32 has been released and https://github.com/gardener/gardener-extension-networking-calico/pull/250 is available as release
+    providerConfig:
+      apiVersion: calico.networking.extensions.gardener.cloud/v1alpha1
+      kind: NetworkConfig
```

Comment:
Remove this. Please reconcile this manifest again against the others on `master` branch

Reply:
We will retest this then using no NetworkConfig because we haven't done it yet and I'm unsure if it will work without